src/lib/mina/actions/offchain-contract.unit-test.ts

```diff
@@ -12,23 +11,22 @@ import assert from 'assert';
 
 const proofsEnabled = true;
 
-const { OffchainState, OffchainStateCommitments } = Experimental;
+const { OffchainState } = Experimental;
 
-const offchainState = OffchainState({
-  accounts: OffchainState.Map(PublicKey, UInt64),
-  totalSupply: OffchainState.Field(UInt64),
-});
+const offchainState = OffchainState(
+  {
+    accounts: OffchainState.Map(PublicKey, UInt64),
+    totalSupply: OffchainState.Field(UInt64),
+  },
+  { logTotalCapacity: 10, maxActionsPerProof: 5 }
+);
 
 class StateProof extends offchainState.Proof {}
 
 // example contract that interacts with offchain state
 
 class ExampleContract extends SmartContract {
-  // TODO could have sugar for this like
-  // @OffchainState.commitment offchainState = OffchainState.Commitment();
-  @state(OffchainStateCommitments) offchainState = State(
-    OffchainStateCommitments.empty()
-  );
+  @state(OffchainState.Commitments) offchainState = offchainState.commitments();
```

Comment:
this looks a bit nicer now, and was necessary because the empty state now depends on the tree size which is part of the offchain state config